react/src/components/calculators/directors/DirectorsForm.tsx

```diff
@@ -52,6 +54,7 @@ export default function DirectorsForm(props: DirectorsFormProps) {
 
   const handleDeleteRow = (e: React.MouseEvent) => {
     e.preventDefault()
+    invalidateResults()
```

Comment:
this should really happen inside `if(activeRowId)` to make sure changes to the table are actually happening before wiping results